src/utils.c

```diff
@@ -621,3 +622,29 @@ gboolean r_update_symlink(const gchar *target, const gchar *name, GError **error
 
 	return TRUE;
 }
+
+gboolean r_syncfs(const gchar *path, GError **error)
```

Comment:
To have methods more consistent, I would propose to add input guards here (`g_return_val_if_fail`..)